src/s3Client.ts

```diff
@@ -1,11 +1,22 @@
 import { S3 } from '@aws-sdk/client-s3';
+import { requireEnv } from 'require-env-variable';
+
+const {
+  S3_ACCESS_KEY_ID,
+  S3_ACCESS_SECRET,
+  S3_ENDPOINT,
+} = requireEnv(
```

Comment:
For my edification: this pattern will break immediately on process start if the vars aren't set / we don't need to also explicitly check for these in `src/index.ts` right?

Reply:
Yes, it will break on process start if any of the variables are not set, and no we do not need to also check in `src/index.ts`.